src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/PlatformLayouts.java

```diff
@@ -141,7 +141,7 @@ private SysV() {
         /**
          * The {@code T*} native type.
          */
-        public static final ValueLayout C_POINTER = ofPointer(LITTLE_ENDIAN, ADDRESS.bitSize());
+        public static final ValueLayout C_POINTER = ofPointer(LITTLE_ENDIAN, 64);
```

Comment:
I somewhat get the change in `Win64()` and `AArch64()`, but why here at `SysV()`? Surely x86_32 is the platform with 32-bit pointers?

Reply:
SysV here is the 64-bit SysV ABI, not 32. Perhaps this needs to be disambiguated yet, but we can cross that bridge when adding 32-bit support.

Any way, I pushed the wrong thing here. This is supposed to go in another PR. Will fix.